Sources/TootSDK/Models/TootFeature.swift

```diff
@@ -1,8 +1,303 @@
+
+        /// Create a requirement for a minimum display version
+        /// - Parameters:
+        ///   - flavour: The server flavour
+        ///   - displayVersion: The minimum display version required
+        public static func from(_ flavour: TootSDKFlavour, displayVersion: String) -> FlavourRequirement {
+            FlavourRequirement(
+                flavour: flavour, minDisplayVersion: Version(tolerant: displayVersion), maxDisplayVersion: nil, minVersion: nil, maxVersion: nil)
+        }
+
+        /// Create a requirement for a display version range
+        /// - Parameters:
+        ///   - flavour: The server flavour
+        ///   - displayVersion: The minimum display version required
+        ///   - to: The maximum display version supported
+        public static func from(_ flavour: TootSDKFlavour, displayVersion: String, to maxDisplayVersion: String) -> FlavourRequirement {
+            FlavourRequirement(
+                flavour: flavour,
+                minDisplayVersion: Version(tolerant: displayVersion),
+                maxDisplayVersion: Version(tolerant: maxDisplayVersion),
+                minVersion: nil,
+                maxVersion: nil
+            )
+        }
+
+        /// Create a requirement for a maximum display version
+        /// - Parameters:
+        ///   - flavour: The server flavour
+        ///   - displayVersion: The maximum display version supported
+        public static func until(_ flavour: TootSDKFlavour, displayVersion: String) -> FlavourRequirement {
+            FlavourRequirement(
+                flavour: flavour, minDisplayVersion: nil, maxDisplayVersion: Version(tolerant: displayVersion), minVersion: nil, maxVersion: nil)
+        }
+
+        // MARK: - Combined Requirements
+
+        /// Create a requirement with both API version and display version fallback
+        /// - Parameters:
+        ///   - flavour: The server flavour
+        ///   - version: The minimum API version required
+        ///   - fallbackDisplayVersion: The minimum display version to use as fallback when API version is not available
+        public static func from(_ flavour: TootSDKFlavour, version: Int, fallbackDisplayVersion: String) -> FlavourRequirement {
+            FlavourRequirement(
+                flavour: flavour,
+                minDisplayVersion: Version(tolerant: fallbackDisplayVersion),
+                maxDisplayVersion: nil,
+                minVersion: version,
+                maxVersion: nil
+            )
+        }
+
+        // MARK: - Legacy Support (Deprecated)
+
+        /// Create a requirement for a version range (legacy method using display versions)
+        /// - Note: This method is deprecated. Use `from(_:displayVersion:to:)` for display versions or `from(_:version:to:)` for API versions
+        @available(*, deprecated, renamed: "from(_:displayVersion:to:)")
+        public static func range(_ flavour: TootSDKFlavour, from: String, to: String) -> FlavourRequirement {
```

Comment:
Is this needed? Version checks are a new feature. No one had a chance to use it.